### PR TITLE
Add "select all toggles" button in consents page

### PR DESCRIPTION
**Pull request by gentymeri**

### CODE REVIEW

File: src/components/Consents.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div>
+    <button @click="selectAllToggles">Select All</button>
```
Comment (dariosalvi78) [inline src/components/Consents.vue:3]: strings literals must always be in the translation files

File: src/components/Consents.vue [hunk 2]
```diff
@@ -109,6 +110,18 @@ export default {
     }
   },
   methods: {
+    selectAllToggles () {
```
Comment (dariosalvi78) [inline src/components/Consents.vue:113]: make sure that all OS level permissions are triggered

**Review by dariosalvi78 — CHANGES_REQUESTED** [review]
there is a problem with this, some of the consent items should trigger the process to obtain OS-level permissions to access APIs. You need to make sure that those permissions are actually triggered when doing "select all".  You can check that in the browser emulation too.

File: src/components/Consents.vue (hunk 2 above)
Comment (gentymeri) [reply src/components/Consents.vue]: pushed a fix for this

**Pull request merged**